catalog-rest-service/src/test/java/org/openmetadata/catalog/resources/EntityResourceTest.java

```diff
@@ -323,9 +330,9 @@ public void afterAllTests() throws Exception {
   public abstract Object createRequest(String name, String description, String displayName, EntityReference owner)
       throws URISyntaxException;
 
-  // Add all possible relationship to check if the entity is missing any of them after deletion
-  public Object addAllRelationships(TestInfo test, Object create) throws HttpResponseException {
-    return create;
+  // Add all possible relationships to check if the entity is missing any of them after deletion
+  public T beforeDeletion(TestInfo test, T entity) throws HttpResponseException {
```

Comment:
`addAllRelationships` seems to have been more informative (based on comment above) more than `beforeDeletion`

May be I'm losing track of the sequence of API calls for tests.

Reply:
`beforeDeletion` tells you when your code will be invoked and not what your code should do that could change in the future.